tests/error_009_missing_js_module.js.out

```diff
@@ -0,0 +1 @@
+NotFound: Cannot resolve module "./bad-module.js" from "[WILDCARD]/tests/error_009_missing_js_module.js"
```

Comment:
Shouldn't we get a line number in the exception?

Reply:
It seems that custom exception gets empty stack trace, especially in ResolveCallback (GetCurrentStackTrace would not help either. I believe this is due to module resolution happening before running the code so there is not any actual stack trace)

Reply:
Ok. I'm doing this big refactor of the module code right now - I'm going to delay landing this and/or work it into my patch...

Reply:
@ry Got it.

Reply:
Changed my mind. I will just land now to avoid this crash - I doubt I will get my module stuff landed in time for the next release. I just anticipate changing this stuff soon.

Thanks for the patch!